microsoft/testsuites/kvm/kvm_unit_tests.py

```diff
@@ -43,5 +44,9 @@ def verify_kvm_unit_tests(
         virtualization_enabled = node.tools[Lscpu].is_virtualization_enabled()
         if not virtualization_enabled:
             raise SkippedException("Virtualization is not enabled in hardware")
+        if not isinstance(node.os, (CBLMariner, Ubuntu)):
+            raise SkippedException(
+                "KVM unit tests supported only for CBLMariner and Ubuntu"
```

Comment:
Is it a hard requirement, or just not implemented for other distros?

Reply:
Just not implemented in lisa.

Reply:
If so, give the clearer message like "not implemented in LISA." If the work is in todo list, add one more sentence like "It needs to be supported in future."

Reply:
Done.